### DISCUSSION TITLE
Rename plugin?

### DISCUSSION THREAD

**Discussion by wsdjeg** [w0rp/ale#4165] [Ideas]
The current version of ale is not only a asynchronous lint engine(ale) for vim or neovim, it also includes code completion, fixing, Go To Definition, Find References, Hovering, Symbol Search, Refactoring: Rename, Actions. so maybe the name should be: Vim Code analytics. or asynchronous code analytics etc.

**Comment by w0rp** [comment]
I actually came up with the name "ALE" first before I decided what the acronym would mean, because I'm teetotal, as a joke. I certainly wouldn't change the name now.

**Comment by w0rp** [comment]
I'm going to propose that you change your name to "Senor Cardgage."
